lms/djangoapps/instructor/utils.py

```diff
@@ -275,10 +272,22 @@ def ora2_data_queries():
 ) AS `feedback`
 FROM `submissions_submission` AS `sub`
 JOIN `submissions_studentitem` AS `student` ON `sub`.`student_item_id`=`student`.`id`
-WHERE `student`.`item_type`="openassessment" AND `student`.`course_id`=%s
+WHERE `student`.`item_type`="openassessment" AND `student`.`course_id`="{1}"
     """
-
-    return RAW_QUERY
+    if include_email == 'False':
```

Comment:
I'd suggest swapping the order of this conditional.  I think that positive conditionals are easier to read than negative ones.  It will also put the query section you want to use to include email addresses right next to the check on include_email's value.